radsat-sk/operation/subsystems/RTransceiver.c

```diff
@@ -281,10 +283,25 @@ int transceiverResetWatchDogs(void) {
 	return error;
 }
 
+/**
+ *  Soft Reset the ISIS TRXVU VU_RC and VU_TC.
+ * @return	Error code; 0 for success, otherwise see hal/errors.h.
+ */
+int transceiverSoftReset(void){
+
+	int error = IsisTrxvu_softReset(TRANSCEIVER_RX_I2C_SLAVE_ADDR);
+
+	//TODO: record errors (if present) to System Manager
+	if (error != 0)
+			return error;
+
+	return 0;
+}
+

```

Comment:
- White space and indentation issues; follow example of other functions in this file
- Instead of providing a slave addr, I believe this `IsisTrxvu_softReset` is asking an _index_; see the `transceiverPowerCycle` function above for an example
- You should "move" this function to be next to the `transceiverPowerCycle` function in this file, just so they're grouped together since they have very similar functionality